File: src/main/resources/scripts/globals.lua
package.path = cache:getAbsolutePath() .. '/?.lua;' .. package.path

class = require 'middleclass'

function registerEventHandler(handler)
    bus:register(scriptingCore, lang, handler);
end

function registerFMLHandler(handler)
    fml:registerModLoadEvents(scriptingCore, lang, handler);
end

function registerMCHandler(handler)
    fml:registerMinecraftEvents(scritpingCore, lang, handler);
end

function registerItem(handler)
    fml:registerItem(scriptingCore, lang, handler);
end

File: src/main/resources/scripts/globals.js
function registerEventHandler(handler){
    bus.register(scriptingCore, lang, handler);
}

function registerFMLHandler(handler){
    fml.registerModLoadEvents(scriptingCore, lang, handler);
}

function registerMCHandler(handler){
    fml.registerMinecraftEvents(scriptingCore, lang, handler);
}

function registerItem(handler){
    fml.registerItem(scriptingCore, lang, handler);
}

function include(path){
    scriptingCore.importPath(path);
}